Mining Pool Rebalance and Loot Reworks, Pt. 1

Sun-Soaked opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
**A number of tweaks, rebalances, additions, ports, and partial removals of assorted mining loot:**
- Immortality Talisman spent some time with the ancient masters to better cultivate it's connection to the null zone. The Talisman can now be held in and triggered in pockets, allows for use of hands and some items while inside the shadow realm, and can set a custom warcry with alt-click!
-To compensate slightly, the active time and cooldown have both been amplified by 1.5.
- The throwing hook now has the ability to do weak, quick attacks in melee range. These are easily defeated by armor, but can be brutal against unprepared targets.
- Adds the Freeze Cube, a strange magical boomerang that freezes targets before bouncing back to the user. Remember to catch on return, unless you want to end up like your opponent.
- The possessed sword acquired via. Necropolis chests now has a custom aesthetic, increased damage, and the ability to slowly bleed up build affects akin to the cleaving saw. Devour your enemies with a friend by your side.
- Most unexpectedly, the cursed katana has received a strange power from an unknown patron! Beware, for the being that now lurks inside hungers for more than just the blood of your enemies.
- The Broodmother Tongue and Legion Spine crusher trophy effects have been significantly buffed, and now enjoy a slightly less useless place within the crusher extended universe.
- Pandora's "Hope" Talisman now provides armor akin to the Bone Talisman, alongside the mood boost.
- The Dark Blessing has been removed from chest loot, may be re-added later in this pr depending on a possible rework.
- The Jacob's ladder has been removed from the loot pool. You know what you did.
- Nobody seems to have actually noticed, but the paradox bag was by far the most robust storage solution in the game, and could contain pretty much everything you'd ever need. It has had it's original storage cap reduced by half- still a metric ton, but no longer enough to carry all of SS13. It also has new prettified sprites and can now fit on belt slots.
- The soulstone item has been replaced with a malfunctioning necrostone- it can resurrect more active entities than the typical stone, but it grants no loyalties.
- The cowhand's gun, Slowpoke, can now be found in the necropolis's treasure caches! This weapon is self-charging, and the sparkling blasts it creates can pierce both rock and flesh. When its cell is low, the power it projects grows considerably.
- A hermit's shawl can now be found in the necropolis's loot- it provides an increase to movement speed, protection from temperature, and a rapidly-recharging magical barrier. However, be not overconfident: When your shield is down, the damage you take from attacks is **considerably** amplified.
- A curiously fashionable outfit has appeared **amoung** the loot of the necropolis...
- Ports the tribal torch from F13, can be crafted from wood in the tribal crafting tab. Use it in hand to plant it in the ground, then hit it with something hot to light it.

## Why It's Good For The Game
Adds some fun new loot to the pool.
Reworks some of the weaker or more unabashedly overpowered items in the necro pool, making the lives of aspiring treasure hunters easier.
Barely begins the long process of redesigning necroloot to both be more stable and more engaging.
Next is probably more nerfs/reworks(enjoy your ERT-level no slowdown armor while it lasts), and a buff to tendrils themselves.

## Changelog
:cl:
add: A particularly drippy outfit. Don't take it off!
add: The ascetic's garb, an overwear item that speeds up the wearer and surrounds them in a protective desert wind. Singificantly increases damage taken when shield is down.
add: Ports the Ice Cube artifact from /TG/
add: Ports spur from Yogstation, reworks it slightly to rebalance it to mining pool standards.
add: Ports tribal torch from F13 and adds the associated crafting recipe
add: New cursed katana sprites and aesthetic.
add: A number of artefact item icons ported from modern /TG/ 
add: Concussion Gauntlets! Special martial art content incoming.
tweak: Replaces soulstone with a nerfed necrostone
tweak: Assigns new sprites to the Paradox bag, and nerfs its storage capacity
tweak: A mining scanner has been added to the diamond pickaxe necroloot drop.
tweak: An envy's knife has been added to the voodoo doll drop. (thematic appropriateness)
balance: The cursed heart's "beat" ability can now be triggered when stunned or cuffed. Bloodloss from missing a beat has been halved. Pending possible nerfs to heal amount if this is too busted.
balance: Redesigned immortality talisman to be more generally useful, and to accept a custom BIG SCARY SHOUT
balance: Redesigns Hook Shot item to apply fast attacks in melee range. 
tweak: Added new pathing to some necro items, and reorganized list to contain two less items (for now)
balance: Removed Jacob's Ladder and Dark Blessing from the necropolis loot pool.
balance: Added buffed rework of possessed blade to necroloot pool, gaining amplified damage, block chance, and bloodletting
balance: Reworks cursed blade to cause acts of hubris and occasionally painful death
balance: Buffed all the elite items from useless to ok, since you can only get them in the first place by fighting a pvp megafauna
tweak: gave the necropolis chest power miner a small chance to drop a chooseable version, instead. Spin the wheel of overwhelmingly low chances.
tweak: removes ability to choose gravokinetic guardian until the game-breaking visual bugs are fixed.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Sun-Soaked committed:
Miners are a threat to society. In today's essay

Contents:
-Loot pool adjusted to remove some currently-broken items, and to add new items.
-Guardian spawner now has a 25% chance to spawn a selectable version. (spin the wheeel)
-Immortality 2 has finally been released, and now it fits into pockets and has a modular funny scream. 
With the power of rebalance, not only can you now use your hands to pray for forgiveness while bubblegum waits patiently outside your void patch, you can also directly use the ability from your hotbar, hands free.
(Cooldown and active time have both been increased by 1.5 of their original value, pending possible increase to 2.0 if abusable.)

Sun-Soaked committed:
master...forgive me. 

Katana more like banana

Sun-Soaked committed:
what is blood even

Sun-Soaked committed:
Time for one line commits(Elite pt.1)

Sun-Soaked committed:
best item to wish for irl (Elite pt. 2)

Sun-Soaked committed:
There's the final one

Sun-Soaked committed:
your brain gets smart but your blessing gets dumb

Sun-Soaked committed:
double negative

Sun-Soaked committed:
guh(I fucked up)

I fucked up the nemesis system, this fixes it(why is this so fucking well coded hnnngh)

ZephyrTFA commented on code/game/objects/items/weaponry.dm:
```diff
@@ -223,13 +223,51 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
+/obj/item/katana/cursed
+	name = "ominous katana"
+	desc = "An ancient single-edged weapon, possessing a flawless sheen despite it's clear age. The blade seems almost too perfect, and passes through most substances like water. <span class='warning'><b>Something is wrong.</b></span>."
+	force = 50
```
Far too much damage, reduce it and make the faction bonus larger

AsciiSquid replied:
I suggest `force = 20` and `faction_bonus_force = 30` to still meet that 50 damage bar while keeping it on-par with other swords.

AsciiSquid commented on code/game/objects/items/weaponry.dm:
```diff
@@ -223,13 +223,51 @@ for further reading, please see: https://github.com/tgstation/tgstation/pull/301
+	var/list/nemesis_factions = list("mining", "boss")
+	var/faction_bonus_force = 15//Even with the highest damage of any practically accessible weapon, it still can't reliably kill some fauna without putting people into crit. Just mining balance I guess
+
+/obj/item/immortality_talisman/examine(mob/user)
```
```suggestion
/obj/item/katana/cursed/examine(mob/user)
```
Assuming that's a typo

Sun-Soaked committed:
katana damage reduced by 15, bonus increased by 10

Sun-Soaked committed:
Yeah that's uhh

Co-authored-by: AsciiSquid <39249137+AsciiSquid@users.noreply.github.com>

Sun-Soaked committed:
Adds the new icons I was drumming up earlier

funny artefact

Sun-Soaked committed:
Paradox Shmaradox

Updated sprotes, slight storage nerf(not as heavy as TG)

Sun-Soaked committed:
fixes paradox sprotes

Sun-Soaked committed:
concussion gauntlet sprotes

Sun-Soaked committed:
basic concussion gauntlets!

This is pretty much a framework, I'm still working on the martial art.

Sun-Soaked committed:
action/effect sprotes

currently unused but for the Earthquake Stance/Cursed Katana powers

Sun-Soaked committed:
minor guardian thing

removes ability to choose gravokinetic guardian(for now)

Adds a special name and desc to the chooseable miner version

Sun-Soaked committed:
the actual new katana sprites

I forgor :skull:

Sun-Soaked committed:
cursed inhands

literally and figuratively

Sun-Soaked committed:
sprote fixes and cursed imp

adds cursed kot sprites to their proper place
fixes my new sprotes to be unshitted

Sun-Soaked committed:
don't lecture me with your 15-dollar code

adds drip item, replaces soulstone with a nerfed necrostone

Sun-Soaked committed:
ice qube and new chains

-adds the freeze cube, freezes on throw and boomerangs back to freeze you.

-redesigns the magic hook to do quick, weak attacks

-drip shoes now make you faster.

Sun-Soaked committed:
adds new sell prices

-adds a few new prices for sale of the new loot

Sun-Soaked commented:
This is ready for testmerging- I need help from more accomplished coders on the Katana mechanics and the Earthquake Stance, but both have a gutted fully-functional version in this code. Everything else should be good to go.

Sun-Soaked committed:
adds Slowpoke!

-Adds an S-Tier necroloot item, Slowpoke. This self-charging "mining" tool is also a particularly robust weapon at medium to close range, growing more powerful as it's clip is depleted.

Sun-Soaked committed:
It was the worst of times, it was the worst of times

please tell me this wasn't what actually caused it

Sun-Soaked committed:
please fix it

Sun-Soaked committed:
if you break this time

-when the spriting DMI is insufferable

Sun-Soaked committed:
literally what is wrong with you

Sun-Soaked committed:
Merge branch 'shiptest-ss13:master' into patch-2

Sun-Soaked commented:
time to see if merging up fixed the funkiness

Sun-Soaked committed:
spin the devil's dice

cmon honey, papa needs a new spur sprite

Sun-Soaked commented:
thank christ

Sun-Soaked committed:
spur balancing

-decreases charge delay relative to shot cost increase. Boost to previously pitiful rate of depleted fire, still bad but no longer unusable

Sun-Soaked committed:
adds the ascetic's garb

-Speed up, quick-shield, massive vulnerability when shield is down. Don't get cocky!

(also fixes a minor list bug)

Sun-Soaked committed:
incredibly slight garb nerf

-Increases charge delay by 1.5, it was a little too powerful before.

Also fixes name

Sun-Soaked committed:
assorted stuff

-Adds new export prices for items
-adds an envy's knife to the voodoo doll drop
-adds an automatic mining scanner to the pickaxe drop

Sun-Soaked committed:
cursed heart buff

-The "pump blood" action is now hands free, meaning that being cablecuffed/stunned is no longer instant permadeath

-You now lose half as much blood on a missed beat.

Sun-Soaked committed:
lava boat buff, polish

-The magic lavaboat can now traverse space tiles at jetpack speed. Set sail!

-slightly reduces power of cursed heart buff.

-splits Envy Knife into its own necrodrop

Sun-Soaked committed:
shielding time

-adds a better shield effect for the garb

-fixes le boatte as well

Sun-Soaked committed:
tax fraud papers

(actually just a super minor QOL thing for the garb)

Sun-Soaked committed:
heart rebuff

-every single person who shipped and tested the new version died horribly, so I'm just gonna

Sun-Soaked committed:
Tribal torch

-ports tribal torch from f13, can be crafted from wood in the tribal tab and planted in the ground for RUSTIC DECORATION HOUR

Sun-Soaked committed:
dm ehh

.dme stuff for tribal torch

Sun-Soaked committed:
I forgor

-tragedy

ZephyrTFA commented on code/datums/martial/earthquake.dm:
```diff
@@ -0,0 +1,152 @@
+	H.physiology.stamina_mod = initial(H.physiology.stamina_mod)
+	H.physiology.stun_mod = initial(H.physiology.stun_mod)
+	H.physiology.pressure_mod = initial(H.physiology.pressure_mod)
+	H.physiology.cold_mod = initial(H.physiology.cold_mod)
```
you forgot to also set heat_mod to intial

Sun-Soaked replied:
I'll fix it RQ, but note this is a WIP version and the proper implementation of EQS will likely be part of my port of newtg's martial art framework

Sun-Soaked committed:
heat fix

five nights at freddies

MarkSuckerberg commented on code/datums/mood_events/generic_positive_events.dm:
```diff
@@ -175,6 +175,18 @@
+	mood_change = 5
+
+/datum/mood_event/nojordans
+	description = "<span class='warning'>They're gone... my fasion is ruined. I can feel my self esteem decaying... </span>\n"
```
```suggestion
	description = "<span class='warning'>They're gone... my fashion is ruined. I can feel my self esteem decaying... </span>\n"
```

MarkSuckerberg commented on code/modules/mob/living/simple_animal/guardian/guardian.dm:
```diff
@@ -746,3 +746,7 @@ GLOBAL_LIST_EMPTY(parasites) //all currently existing/living guardians
 	random = FALSE
+	name = "glimmering shard"
+	desc = "Seems to be a very old rock, may have originated from a strange meteor. <b>This one looks exceptionally pure.</b>"
+	possible_guardians = list("Assassin", "Chaos", "Charger", "Dextrous", "Explosive", "Lightning", "Protector", "Ranged", "Standard", "Support	")
```
```suggestion
	possible_guardians = list("Assassin", "Chaos", "Charger", "Dextrous", "Explosive", "Lightning", "Protector", "Ranged", "Standard", "Support")
```

MarkSuckerberg reviewed (approved):
This is some good work. 👍👍👍

Sun-Soaked committed:
Update code/datums/mood_events/generic_positive_events.dm

Co-authored-by: Mark Suckerberg <29362068+MarkSuckerberg@users.noreply.github.com>

Sun-Soaked committed:
Update code/modules/mob/living/simple_animal/guardian/guardian.dm

Co-authored-by: Mark Suckerberg <29362068+MarkSuckerberg@users.noreply.github.com>
